hudi-common/src/main/java/org/apache/hudi/common/config/ConfigGroups.java

```diff
@@ -32,6 +32,7 @@ public enum Names {
     SPARK_DATASOURCE("Spark Datasource Configs"),
     FLINK_SQL("Flink Sql Configs"),
     WRITE_CLIENT("Write Client Configs"),
+    METADATA_SYNC("Metadata Sync Configs"),
```

Comment:
to be accurate we should call it meta_sync or catalog_sync

Reply:
Makes sense.  Changed to `META_SYNC` to be consistent with config naming.